rust/tw_proto/tests/proto_ffi_tests.rs

```diff
@@ -0,0 +1,38 @@
+use tw_memory::ffi::{c_byte_array::CByteArray, free_string};
+
+/// Test `pass_eth_signing_msg_through` to avoid dropping code coverage.
+#[test]
+fn test_pass_eth_signing_msg_through() {
+    let serialized = vec![
```

Comment:
Makes sense. I wanted to leave the vector because it can be easily split into multiple lines via `rustfmt`, but the hex string should be split manually via `\` character. Will do

Reply:
I placed the hex string without the break lines.
Done